src/Model/BookManager.php

```diff
@@ -82,4 +81,16 @@ public function update(Book $book)
             return self::DATABASE_ERROR;
         }
     }
+
+    public function delete($id)
+    {
+        $statement = $this->pdo->prepare("DELETE FROM " . self::TABLE . " WHERE id=:id");
+        if ($statement == false) {
+            return self::DATABASE_ERROR;
+        }
+        if ($statement->bindValue('id', $id, \PDO::PARAM_INT) == false) {
+            return self::DATABASE_ERROR;
+        }
+        $statement->execute();
```

Comment:
You should specify your function return type (boolean probably)
And you should return in every cases.
In fact PDOStatement::execute() also return a boolean. May be you should return it at the end.